src/main/java/com/qdesrame/openapi/diff/compare/SchemaDiff.java

```diff
@@ -77,7 +77,7 @@ public static SchemaDiffResult getSchemaDiffResult(
     }
   }
 
-  protected static Schema resolveComposedSchema(Components components, Schema schema) {
+  public static Schema resolveComposedSchema(Components components, Schema schema) {
```

Comment:
Don't know how to make this work without doing this... I know this isn't the most elegant thing to do, but it is a utility method and making it public isn't harmful imo